shop_clothes/hw_shop_clothes/styles/contact_us.css

```diff
@@ -97,3 +97,20 @@ form input[type="submit"]:hover {
     background-color: rgba(340, 340, 340);
     cursor: pointer;
 }
+
+@media screen and (max-width: 1309px) {
```

Comment:
why so big value?
you have a place to fit 3 blocks in one row for smaller screens